wagtail/images/models.py

```diff
@@ -303,7 +308,23 @@ def get_rendition(self, filter):
             )
         except Rendition.DoesNotExist:
             # Generate the rendition image
-            generated_image = filter.run(self, BytesIO())
+            try:
+                LOGGER.debug("Generating '%s' rendition for image %d", (
```

Comment:
I'm not entirely sure if this log before is necessary.

My main intention is to cover the case where generating an image uses so much memory that the application either locks up or is OOM killed (or platform-specific kill). I suspect an exception handler probably wouldn't catch that.

Question is, is it important enough to have 2 log messages for each generation as opposed to just 1?